README.md

```diff
@@ -105,7 +105,11 @@ kubectl apply -f https://raw.githubusercontent.com/<org>/etcd-operator/<tag or b
 
 ## Contributing
 
-// TODO(user): Add detailed information on how you would like others to contribute to this project
+See [CONTRIBUTING](CONTRIBUTING.md) for details on setting up your development environment, submitting patches and the contribution workflow.
+
+Please refer to community-membership.md for information on becoming an etcd project member. We welcome and look forward to your contributions to the project!
```

Comment:
Where is community-membership.md? Do you mean https://github.com/etcd-io/etcd/blob/main/Documentation/contributor-guide/community-membership.md?

Reply:
Might mean Kubernetes community management.